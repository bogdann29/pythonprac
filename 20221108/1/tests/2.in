a = DivStr("bobao")
b = DivStr()
print(b%2)
print(a % 4)
print(* a % 6)
print(a % 5)
print(* a // 5)