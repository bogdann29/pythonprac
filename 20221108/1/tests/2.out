
o
b o b a o
bobao
b o b a o
